.gitignore

```diff
@@ -1 +1,6 @@
-*.pyc
\ No newline at end of file
+# Python Build Artifacts
+*.pyc
+
+# Virtual Environments
+.venv/
+venv/
```

Comment:
hey Chris, I do think people tend to ignore the venv files, but in this DevNet Express Issue ([19](https://github.com/CiscoDevNet/devnet-express-dna-issues/issues/19)) I wondered if you mean to leave it in?